src/routes/NavBar.svelte

```diff
@@ -176,3 +176,12 @@
     </div>
   </div>
 </div>
+
+<style>
+  .glass {
+    background: hsl(var(--b2) / 0.6);
+    /* box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1); */
+    backdrop-filter: blur(10.5px);
+    -webkit-backdrop-filter: blur(10.5px);
+  }
+</style>
```

Comment:
Similarly, all these could be written as Tailwind classes (also removing the commented out line): `bg-<some color from our theme>` (see tailwind.config.js or just rely on Tailwind IntelliSense) and `filter backdrop-blur`.